praw/models/reddit/rules.py

```diff
@@ -0,0 +1,296 @@
+"""Provide the Reason class."""
+from json import loads
+from typing import Dict, Generator, List, Optional, TypeVar, Union
+from urllib.parse import quote
+
+from ...const import API_PATH
+from ...exceptions import ClientException
+from .base import RedditBase
+
+_Rule = TypeVar("_Rule")
+Reddit = TypeVar("Reddit")
+Subreddit = TypeVar("Subreddit")
+
+
+class Rule(RedditBase):
+    """An individual Rule object.
+
+    **Typical Attributes**
+
+    This table describes attributes that typically belong to objects of this
+    class. Since attributes are dynamically provided (see
+    :ref:`determine-available-attributes-of-an-object`), there is not a
+    guarantee that these attributes will always be present, nor is this list
+    necessarily comprehensive.
+
+    ======================= ===================================================
+    Attribute               Description
+    ======================= ===================================================
+    ``created_utc``         Time the rule was created, represented in
+                            `Unix Time`_.
+    ``description``         The description of the rule, if provided, otherwise
+                            a blank string.
+    ``kind``                The kind of rule. Can be `link`, `comment`,
+                            or `all`.
+    ``priority``            Represents where the rule is ranked. For example,
+                            the first rule is at priority ``0``.
```

Comment:
It's an index on Reddit's rule list, indicating the order of a rule. It's identical to rules in a list, but it can only be changed by SubredditRules.reorder. Each priority is unique in theory, but a call to reorder could change that.

Reply:
In that case, it would be nice to mention that the priority is the index into the list.